src/index.ts

```diff
@@ -66,6 +66,9 @@ export {
   ProgressBar,
   ProgressBarList,
   RenderHtmlWrapper,
+  imgRenderer,
+  olRenderer,
+  ulRenderer,
```

Comment:
Waarom is opnieuw exporteren op een andere plek nodig? Kun je in de app niet
```ts
import RenderHtmlWrapper, { imgRenderer, olRenderer, ulRenderer } from '@observation.org/react-native-components'
```
doen?

Reply:
Ja, je hebt gelijk. Deze import werkt vermoedelijk wel:

```ts
import RenderHtml, { imgRenderer, olRenderer, ulRenderer } from '@observation.org/react-native-components/src/components/RenderHtmlWrapper'
```

Nadeel is dat we de interne structuur van de component-library gebruiken (`src/components`). In de huidige setup exporteren we alle componenten als named exports vanaf de root. We verliezen de namespaces die we eerst wel hadden. Alle exports komen in 1 lange lijst van named exports terecht. Wellicht is er een betere oplossing mogelijk. Maar dat doen we niet in deze PR, dus voor nu is het toevoegen van exports in de root van de library wel goed.